### PR TITLE
Feature/experimental-dynamic-data

### PR DESCRIPTION
**Pull request by fsher**
* initiate Action Container with checking supportability state
* implement dynamic data
* fix react root unmounting for twtiter

### CODE REVIEW

File: src/ui/ActionContainer.tsx [hunk 1]
```diff
@@ -273,49 +274,92 @@ export const ActionContainer = ({
...
+        // if after refresh user clicked started execution, we should not update the action
+        if (executionState.status === 'idle') {
+          setAction(newAction);
+        }
+      } catch (e) {
+        console.error(
+          `[@dialectlabs/blinks] Failed to fetch dynamic data for action ${action.url}`,
+        );
+        // if fetch failed, we retry after the same delay
+        timeout = setTimeout(fetcher, dynamicDataConfig.delayMs);
```
Comment (tsmbl) [inline src/ui/ActionContainer.tsx:328]: suggestion - wdyt about setting a soft limit on max update freq to avoid possible abuse? e.g. 1 sec

Comment (fsher) [reply src/ui/ActionContainer.tsx:328]: Good call! Let's do min 1s

Comment (fsher) [reply src/ui/ActionContainer.tsx:328]: Adjusted in `Action` class

**Review by tsmbl — APPROVED** [review]
lgtm!

**Pull request merged**